hooks/useTheme.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+
+// Define the types for themes
+type Theme = "light" | "dark" | "system";
+
+export default function useTheme() {
```

Comment:
this file should be removed as it's not used. Similar logic is already in settings-provider.

Reply:
I'll try to resolve it